### PR TITLE
HTTP library

### PR DESCRIPTION
**Pull request by st0012**
Add `Net::HTTP` class and `URI` module

### CODE REVIEW

File: vm/http.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	for _, m := range builtinHTTPClassMethods {
+		http.ClassMethods.set(m.Name, m)
+	}
+
+	vm.constants["Net"] = &Pointer{Target: net}
+}
+
+var builtinHTTPClassMethods = []*BuiltInMethod{
+	{
+		Name: "get",
```
Comment (janczer) [inline vm/http.go]: Maybe should add description to methods?

Comment (st0012) [inline vm/http.go]: I'll add it after I finish them 😄

**Comment by st0012** [comment]
@adlerhsieh I agree that it seems to be redundant. But since most languages (at least in Go and Ruby)  has this namespace, I think maybe we just haven't know the reason, or that just became a convention.

**Comment by adlerhsieh** [comment]
Yes, I agree.

**Comment by st0012** [comment]
I found that `Net::HTTP` library in ruby is far more complicated than I thought before, so I'll merge this PR first although it only adds `GET` method. I think to complete the hole http library requires several PRs.

**Comment by st0012** [comment]
I'll open an issue to list what's lack for the http lib, cc @adlerhsieh @janczer

**Pull request merged**